public/data/stat-types.json

```diff
@@ -0,0 +1,19 @@
+{
+    "StatTypes": [
```

Comment:
any reason to not use the `Stat` value as the key here? It seems like you could reduce the lookup complexity by doing that and still have access to everything you need.

this top level key is just making your life harder when you are going to do lookups for a particular stat.